### PR TITLE
#6897: add page editor link to template gallery

### LINKED ISSUE #6897: Add Start from Template link in Page Editor
## User Story

As a mod creator, I start with a template, so that I can build my use case faster/more successfully

## Motivation

- We show templates on the website/welcome page. But there's currently no affordance in the Page Editor

## Acceptance Criteria

- The "Add" dropdown in the Page Editor has a link to the template gallery labeled "Start with a Template"
- Is after the starter bricks, with a horizontal rule separating the stater bricks and templates link
- Clicking the link redirects the current page to the template gallery
- @BrandonPxBx did we want to use the external link icon there?: https://fontawesome.com/v5/icons/external-link-alt?f=classic&s=solid

## Design

image

### PR DESCRIPTION
**Pull request by twschiller**
## What does this PR do?

- Closes #6897 
- Adds a link to the template gallery to the "Add" dropdown
- Adds a `PageEditorViewTemplates` event to track the button being clicked

## Discussion

- We didn't have a way to redirect the tab from the Page Editor, so I added a method to navigate the inspected tab

## Demo

image

## Checklist

- [ ] Add tests: TODO add a Rainforest QA test because the changes crosses boundaries
- [x] New files added to `src/tsconfig.strictNullChecks.json` (if possible)
- [x] Designate a primary reviewer: @mnholtz

### CODE REVIEW

**Review by BrandonPxBx — APPROVED** [review]
Looks good 👍

File: src/pageEditor/sidebar/AddStarterBrickButton.tsx [hunk 1]
```diff
@@ -52,55 +58,67 @@ const DropdownEntry: React.FunctionComponent<{
...
       {entries}
+
+      <Dropdown.Divider />
+      <Dropdown.Item
+        onClick={() => {
+          reportEvent(Events.PAGE_EDITOR_VIEW_TEMPLATES, {
+            sessionId,
+          });
+          navigateTab(thisTab, {
+            url: "https://www.pixiebrix.com/templates-gallery?utm_source=pixiebrix&utm_medium=page_editor",
```
Comment (twschiller) [inline src/pageEditor/sidebar/AddStarterBrickButton.tsx:113]: @brittanyjoiner15 let me know if we should be using other/additional utm tags here

Comment (brittanyjoiner15) [reply src/pageEditor/sidebar/AddStarterBrickButton.tsx:113]: @twschiller i've heard that you're technically supposed to use `utm_campaign` as well, so if maybe we want to include `utm_campaign=starter_brick_menu`
